packages/patternlab/styleguide/source/assets/js/templates/eventListingRow.html

```diff
@@ -17,12 +17,12 @@
     {{/if}}
     <div class="ma__event-teaser__content">
       {{#if title.text}}
-      <h3 class="ma__event-teaser__title">
+      <h2 class="ma__event-teaser__title">
```

Comment:
Shouldn't this be h3? I'm seeing this as h2 
<img width="1131" alt="Screen Shot 2022-05-26 at 12 46 22 PM" src="https://user-images.githubusercontent.com/5789411/170535124-ab7c5c0e-c8fe-4f12-a6ba-9295e26b5117.png">

Reply:
You're right.  I missed the result title. The teaser title is back to `h3`.